### PR TITLE
Mobile and Desktop views of HOME & ABOUT page

### PR DESCRIPTION
**Pull request by Beardless-sheik**
- Created HTML markup for 2 pages: Home & About
- Added 2 stylesheets for each of the 2 pages
- Added the speaker section dynamically from Javascript
- Used media queries to alternate the view of the 2 pages at two different breakpoints
- Added a mobile menu to each of the two "mobile sections"
- Deployed this page on Github and added it to the README for reviewers to take a look 
- Made a video of my work on Loom. Link [here](https://www.loom.com/share/2656dabcda184c8c9aff16583c8fd4d7)

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @Beardless-sheik,

You've done pretty well so far!
- Github flow used.
- Correct linters config rules used.
- Good readme.
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- [ ] Add background on images in featured-speakers section.
- [ ] Fix disproportionate images on tablet screen.
- [ ] Handle content overflow on tablet screen.
- [ ] Add Partners section on mobile screen.


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        <a id="join-conference" class="join-button p-2" href="#" role="button"> JOIN THE NKWASHI CONFERENCE </a>
+      </div>
+    </section>
+
+    <section class="speakers">
+      <h2 class="text-center speaker-title"> Featured Speakers</h2>
+      <hr class="orange-dividing-line-about mx-auto">
+      <div id="speakers" class="grid-speaker-container">
+      </div>
+      <button id="see-less-button" class="see-less-button-hide speakers-button rounded p-2" type="button"> SEE LESS <i class="fas fa-chevron-down see-more-speakers-icon>"></i> </button>
```
Comment (mikenath223) [inline index.html:125]: - [ ] Kindly add a background image to each of the images in the featured-speakers section.

| Orignal design | Yours |
| ------------- | ------ |
|  |  |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            </p>
+          </div>
+        </div>
+        <a id="join-conference" class="join-button p-2" href="#" role="button"> JOIN THE NKWASHI CONFERENCE </a>
+      </div>
+    </section>
+
+    <section class="speakers">
+      <h2 class="text-center speaker-title"> Featured Speakers</h2>
+      <hr class="orange-dividing-line-about mx-auto">
```
Comment (mikenath223) [inline index.html:122]: - [ ] Please fix the images on featured-speakers section. On tablet screen(768px) they look disproportionate.
**Screenshot. **👇

File: index.html [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+            </div>
+            <div class="program-description-text">
+              Listen to the speakers from various countries about the messages on community building and innovation
+            </div>
+          </div>
+          <div class="grid-program-container">
+              <img src="./images/exhibition.png" class="program-image-icons" alt="illustration showing an exhibition">
+            <div class="program-description-title">
+              <p>CC Exhibition </p>
+            </div>
```
Comment (mikenath223) [inline index.html:83]: - [ ] On tablet screen(768px), the contents in Main Program section overflows the container. Consider dividing items into more rows on tablet screen.
**Screenshot** 👇

File: index.html [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+    <section class="mobile-hidden partners text-center py-3">
+      <h2> Partners </h2>
+      <hr class="orange-dividing-line-about mx-auto">
+      <div class="partner-image-grid m-4">
+          <img class="partner-logo-airbnb" alt="airbnb logo" src="./images/Airbnb-logo.png">
+          <img class="partner-logo-firefox" alt="firefox logo" src="./images/firefoxLogo.png">
+          <img class="partner-logo-google" alt="google logo" src="./images/Google-logo.png">
+      </div>
+    </section>
```
Comment (mikenath223) [inline index.html]: - [ ] The partners section is hidden on mobile screen kindly add it.
**Screenshot** 👇

**Review by smcommits — APPROVED** [review]
### Status: Approved 👍🏽 

Hi @Beardless-sheik, 

## You have done a great job! Nice work with implementing the changes. Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**